src/components/seances/SeancesCard.vue

```diff
@@ -1,15 +1,27 @@
 <script setup lang="ts">
+import { useRouter } from "vue-router";
+import { useBookingStore } from "@/stores/booking";
+import { getSeanceById } from "@/api/seancesService";
+
 import lengthToTime from "@/helpers/lengthToTime";
-import BaseButton from "../base/BaseButton.vue";
-import type { Movie } from "@/types/movie";
-import type { Seance } from "@/types/seance";
-withDefaults(
-  defineProps<{
-    movie: Movie | undefined;
-    seances: Seance[];
-  }>(),
-  {}
-);
+import BaseButton from "@/components/base/BaseButton.vue";
+import NoResults from "@/components/NoResults.vue";
+import type { MovieWithSeances, Seance } from "@/types/seance";
+const router = useRouter();
+const bookingStore = useBookingStore();
+
+defineProps<{
+  movie: MovieWithSeances;
+}>();
+
+const handleBooking = async (seance: Seance) => {
+  bookingStore.selectedSeance = seance;
+  bookingStore.selectedReservation = await getSeanceById(seance.id);
+  bookingStore.selectedTickets = [];
```

Comment:
this looks like something that could be a `clearSelectedTickets` action in pinia store